src/main/java/frc/robot/OI.java

```diff
@@ -18,6 +18,13 @@ public class OI {
         private static final int kIntakeButtonID = 3; //3 Button, run intake
         private static final int kOuttakeButtonID = 4; //4 Button, run outtake
 
+        private static final int kParkButtonID = 5; //left bumper, park the robot
+        /*for reference "button" 5 is the left bumper
+         * "button" 6 is the right bumper
+         * "button" 7 is the left trigger
+         * and "button" 8 is the right trigger
+        */
+
```

Comment:
Delete this comment, it's written on notion

Reply:
Specifically asked for it to be in a comment.  As we add more buttons, want it to be easier to keep comments.